ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/IdentityProviderConfigServiceIT.scala

```diff
@@ -0,0 +1,489 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.testtool.suites.v1_8
+
+import com.daml.error.ErrorCode
+import com.daml.error.definitions.LedgerApiErrors
+
+import java.util.UUID
+import com.daml.ledger.api.testtool.infrastructure.Allocation._
+import com.daml.ledger.api.testtool.infrastructure.Assertions._
+import com.daml.ledger.api.v1.admin.identity_provider_config_service.{
+  CreateIdentityProviderConfigRequest,
+  DeleteIdentityProviderConfigRequest,
+  DeleteIdentityProviderConfigResponse,
+  GetIdentityProviderConfigRequest,
+  IdentityProviderConfig,
+  UpdateIdentityProviderConfigRequest,
+}
+import com.google.protobuf.field_mask.FieldMask
+
+import scala.concurrent.Future
+
+class IdentityProviderConfigServiceIT extends UserManagementServiceITBase {
+
+  test(
+    "IdentityProviderCreateConfigInvalidArguments",
```

Comment:
I think prefixing each test case with `IdentityProvider` is not needed as they are in the `IdentityProviderConfigServiceIT` test